app/src/app/components/Toolbar/Settings.tsx

```diff
@@ -135,40 +136,44 @@ export const ToolSettings: React.FC = () => {
             Customize district colors
           </Button>
         </CheckboxGroup.Root>
-        <Heading as="h3" weight="bold" size="3">
-          Boundaries
-        </Heading>
-        <CheckboxGroup.Root
-          name="contextualLayers"
-          value={[
-            mapOptions.showCountyBoundaries === true ? 'showCountyBoundaries' : '',
-            mapOptions.prominentCountyNames === true ? 'prominentCountyNames' : '',
-          ]}
-        >
-          <CheckboxGroup.Item
-            value="showCountyBoundaries"
-            onClick={() =>
-              setMapOptions({
-                showCountyBoundaries: !mapOptions.showCountyBoundaries,
-              })
-            }
-          >
-            Show county boundaries
-          </CheckboxGroup.Item>
-          <CheckboxGroup.Item
-            value="prominentCountyNames"
-            onClick={() =>
-              setMapOptions({
-                prominentCountyNames: !mapOptions.prominentCountyNames,
-              })
-            }
-          >
-            Emphasize county names
-          </CheckboxGroup.Item>
-          <CheckboxGroup.Item value="2" disabled>
-            Show tribes and communities
-          </CheckboxGroup.Item>
-        </CheckboxGroup.Root>
+        {mapType !== 'local' ? (
```

Comment:
Nit: Small preference for `!(mapType === 'local') && ...` syntax (or `!!` syntax when looking for true) which has the same safety of a ternary operator but doesn't have the trailing `null` fallback

Reply:
:+1: